libfwupdplugin/fu-efi-section.c

```diff
@@ -324,6 +324,17 @@ fu_efi_section_parse(FuFirmware *firmware,
 			g_prefix_error(error, "failed to parse lzma section: ");
 			return FALSE;
 		}
+	} else if (priv->type == FU_EFI_SECTION_TYPE_GUID_DEFINED &&
+		   g_strcmp0(fu_firmware_get_id(firmware), FU_EFI_SECTION_GUID_TIANO_COMPRESS) ==
+		       0) {
+		g_error("MOO");
+		if (!fu_efi_section_parse_compression_sections(self,
+							       partial_stream,
+							       flags,
+							       error)) {
+			g_prefix_error(error, "failed to parse lzma section: ");
+			return FALSE;
+		}
```

Comment:
huh, what's this?

Reply:
evidence that it's time to go to bed. fixed!